openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1206,3 +1214,78 @@ def validate_glusterfsd_memory_usage(self, size_limit):
                     "Failed memory used  of glusterfsd {} is greater than the"
                     " expected size {} for node/pod {}".format(
                         memory_used, size_limit, gluster_node))
+
+    def heketi_volumes_in_batch(
+            self, heketi_client_node, heketi_server_url, vol_count, size=1,
+            batch_amount=8, timeout=300, interval=5):
```

Comment:
Unused variables `timeout` and `interval`

Reply:
It will be used